src/DynamoCoreWpf/Controls/RunSettingsControl.xaml

```diff
@@ -66,35 +54,57 @@
                     <DataTemplate>
                         <Label Content="{Binding Name}" 
                                IsEnabled="{Binding Enabled}" 
-                               Foreground="#bbbbbb" 
                                ToolTip="{Binding ToolTipText}"
-                               ToolTipService.ShowOnDisabled="True" />
+                               ToolTipService.ShowOnDisabled="True">
+                            <Label.Style>
+                                <Style TargetType="Label">
+                                    <Style.Triggers>
+                                        <DataTrigger Binding="{Binding Enabled}" Value="True">
+                                            <Setter Property="Foreground" Value="#bbbbbb"></Setter>
+                                        </DataTrigger>
+                                        <DataTrigger Binding="{Binding Enabled}" Value="False">
+                                            <Setter Property="Foreground" Value="#444444"></Setter>
+                                        </DataTrigger>
+                                    </Style.Triggers>
+                                </Style>
+                            </Label.Style>
+                        </Label>
                     </DataTemplate>
                 </ComboBox.ItemTemplate>
             </ComboBox>
 
+            <Button Name="RunButton"
+                                Width="100"
+                                Content="{x:Static p:Resources.DynamoViewRunButton}"
+                                Style="{StaticResource ResourceKey=STextButton}"
+                                Click="RunButton_OnClick"
+                                Command="{Binding RunExpressionCommand}"
+                                CommandParameter="{Binding Path=RunInDebug}"
+                                ToolTip="{Binding RunButtonToolTip}"
+                                ToolTipService.ShowOnDisabled="True"
+                                IsEnabled="{Binding Path=RunButtonEnabled}"
+                                Visibility="{Binding RunButtonVisibility}"
+                                Focusable="False" Margin="0,0,2.5,0"/>
+            
             <TextBox Name="RunPeriodTextBox" 
                     Text="{Binding RunPeriod, Converter={StaticResource RunPeriodConverter}, UpdateSourceTrigger=Explicit}"
                      VerticalContentAlignment="Center"
                      Visibility="{Binding RunPeriodInputVisibility}"
                      Width="80" FontSize="14px"
-                     Style="{DynamicResource ResourceKey=SDarkTextBox}" Foreground="#bbbbbb" 
+                     Style="{DynamicResource ResourceKey=SDarkTextBox}" 
+                     Foreground="#bbbbbb" 
                      KeyDown="UIElement_OnKeyDown" Margin="2.5,0,0,0"/>
 
-            <!--<Button Name="cancelButton"
```

Comment:
Really?  I thought @aparajit-pratap did a good amount of work to allow a cancellation.